appveyor.yml

```diff
@@ -16,31 +16,48 @@ skip_commits:
     message: /\[ci skip\]/
 
 cache:
-    - c:\php -> appveyor.yml
+    - C:\ProgramData\chocolatey\bin -> appveyor.yml
+    - C:\ProgramData\chocolatey\lib -> appveyor.yml
+    - C:\tools\php -> appveyor.yml
+    - C:\tools\composer -> appveyor.yml
+    - '%LOCALAPPDATA%\Composer\files -> composer.json'
```

Comment:
no need to expire them when the composer.json is changing. Even if we change one dependency, the others could still benefit from the cache.